FoodSuggestions/app/src/main/res/layout/fragment_instruction.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".fragments.InstructionFragment">
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/idListInstructions"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_marginStart="1dp"
```

Comment:
When all margins are equal, we can use the "android:margin" attribute.

Reply:
Fixed